src/pages/Timeline.js

```diff
@@ -183,10 +184,24 @@ const Timeline = ({
   );
   otherProposals.sort((a, b) => b.end_timestamp - a.end_timestamp);
 
+  const { show } = useModal();
+
   return (
     <Fragment>
       <MigrationNotificationBanner />
       <VoterStatus signaling={signaling} legacy={true} />
+      <Button
+        p="s"
+        css={{ cursor: 'pointer' }}
+        onClick={() =>
+          show({
+            modalType: 'dsrdeposit',
```

Comment:
I think this is why you were getting an error – should be `modaltest`.

Reply:
🤦